interface/main/tabs/templates/patient_data_template.php

```diff
@@ -81,16 +91,18 @@
                     (<span data-bind="text:encounterArray().length"></span>)
                     <span class="caret"></span>
                 </button>
-                <ul class="dropdown-menu" aria-labelledby="pastEncounters">
+                <ul class="container dropdown-menu" aria-labelledby="pastEncounters" style="max-width: 400px;">
                     <!-- ko foreach:encounterArray -->
-                    <li style="display: inline-flex;">
-                        <a href="#" data-bind="click:chooseEncounterEvent">
-                            <span data-bind="text:date"></span>
+                    <li class="row text-nowrap">
+                        <a href="#" class="col-sm-10 enc-sel" data-bind="click:chooseEncounterEvent">
+                            <span data-bind="text:date"></span>&nbsp;-&nbsp;
                             <span data-bind="text:category"></span>
                         </a>
-                        <a href="#" data-bind="click:reviewEncounterEvent">
-                            <i class="fa fa-rotate-left"></i>&nbsp;<?php echo xlt("Review");?>
-                        </a>
+                        <span style="float: right;padding-right: 20px;">
+                            <a href="#" class="btn btn-xs btn-default" data-bind="click:reviewEncounterEvent" 
+                                title='<?php echo xlt("Review");?>'>
+                            <i class="fa fa-search"></i>
```

Comment:
It's better to avoid repeating same element and/or text on every row.  

More importantly from functional perspective, the list is at best misleading - it does not guide users about the encounters they can edit vs those they can only view.  

Our current thinking is to make active encounter be dropdown for open (not locked), editable encounters.  Then current past encounters list will be truly 'View' without need for any buttons.

Will withdraw this PR for now until that change is rolled out.